src/keyword-value.js

```diff
@@ -22,7 +22,7 @@
     this.cssString = value;
   }
 
-  KeywordValue.StyleValueKeyword = ['initial', 'inherit', 'revert', 'unset'];
+  KeywordValue.StyleValueKeyword = ['auto', 'initial', 'inherit', 'revert', 'unset'];
```

Comment:
> It's probably still worth discussion of what the structure should look like for something like "speech-rate" which takes
> 
> | x-slow | slow | medium | fast | x-fast | faster | slower | inherit
> 
> since it doesn't look like there should be a SpeechRateValue class...

I think having a SpeechRateValue class just for the purpose of basing a  SpeechRateKeywordValue class would be OK. But there are many things you could do; we should discuss it.